.gitignore

```diff
@@ -13,3 +13,4 @@ CMakeSettings.json
 out/
 .vs/
 build/
+.vscode
```

Comment:
Don't you need the trailing slash like the other entries?

Reply:
You dont have to but the / just shows that its a folder which helps for clarity so ill add it